packages/notifi-core/lib/operations/CreateDiscordTarget.ts

```diff
@@ -0,0 +1,28 @@
+import { DiscordTarget, Operation } from '../models';
+
+/**
+ * Input param for creating an DiscordTarget
+ *
+ * @remarks
+ * This describes the values to use for email target
```

Comment:
Might want to change it to `Discord` target

Reply:
great catch